app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author_id, class_name: 'User'
```

Comment:
- Good job done so far, but this should be `author` instead of `author_id`. Please fix this to prevent your code from having errors in future milestones :smile: